jest.config.js

```diff
@@ -15,6 +15,8 @@ module.exports = {
     __TEST__: true,
   },
   moduleNameMapper: {
+    '^@algolia/autocomplete-shared/dist/esm/(.*)$':
+      '<rootDir>/packages/autocomplete-shared/src/$1',
```

Comment:
We need to map types exported from `autocomplete-shared/dist/esm` to source types for tests to work properly with Jest.